_docs/developer/developing_the_php_site/modules.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
We don't have documentation on generally writing JS for Submitty right? Like what goes into `site/public/js` and how to load it vs `site/ts`, etc.?

Reply:
I don't think so, I added a blurb on the `addInternal` functions